docs/design/versioning.md

```diff
@@ -34,6 +34,8 @@ Documentation for other releases can be found at
 
 # Kubernetes API and Release Versioning
 
+Reference: http://semver.org
```

Comment:
Generally we prefer links formatted:

```
Reference: [Semantic Versioning](http://semver.org)
```

will become

Reference: [Semantic Versioning](http://semver.org)